data/persons.txt

```diff
@@ -1850,3 +1850,187 @@ ship "Modified Ladybug"
+			"Comma here. Comma doesn't go there. I think that one needs a comma. Hold on, what was a comma again?"
+			"Unidentified sentence appears to be lacking a comma. Very dangerous, and highly likely to incite multiple suggestions on where the comma should go. Calling backup reviewers."
+			"Commas are your greatest friends."
+			"Commas are your greatest enemies."
+
+#		roadrunner56
+			"More semicolons!"
+			"...I think I broke something."
+			"Canadian spelling fails me again."
+			"This is my last review, I swear."
+			"Looks like school's starting back up again. See you in a few months."
+			"Typo detected. Initiating weapons engagement."
+			"Code non-functional. Destroy them."
+
+#		EjoThims
+	phrase
+		word
+			"If you want your pull request not to be merged"
+			"If you prefer that your pull request never be merged"
+			"If you prefer that your pull request sits idle"
+			"If you want your pull request to be closed"
+		word
+			", "
+		word
+			"make sure you keep adding content to it"
+			"always change unrelated things"
+			"do not ever test it and don't add integration tests. Whatever those even are, they're apparently important"
+			"try your best to avoid respecting existing syntax and conventions"
+			"just forget about it for a while"
+			"use the forbidden word: 'realism'"
+			"bury the new feature in semi-relevant data changes"
+			"do not, under any circumstances, ever be willing to compromise"
+			"make sure to completely ignore what the reviewers tell you to do"
+			"skip over writing an issue about it first"
+		word
+			"!"
+			"."
+			". Trust me! It works."
+			". You shouldn't do those things, though! I'm just kidding."
+
+	phrase
+		word
+			"You don't have to be a Reviewer to review pull requests"
+			"Anyone can review a pull request, even you"
+			"If you're unsure of how to help, try reviewing some pull requests"
+			"We welcome an alternative view to the reviews on the pull requests"
+			"If you spot a typo, do start a review"
+			"If you have an idea to make something better, do start a review"
+		word
+			"."
+			"!"
+
+	ship "Waverider" "Proofreader's Pain"
+		"never disabled"
+	ship "Waverider" "Art Critics' Regret"
+		"never disabled"
+	ship "Waverider" "Code Herders' Curse"
+		"never disabled"
+		add attributes
+			"required crew" 2
+	ship "Waverider" "Choreographers' Tears"
+		"never disabled"
+	ship "Waverider" "Adjuncators' Blood"
+		"never disabled"
+		outfits
+			"Particle Cannon" 4
+			"S-970 Regenerator"
+			"Dwarf Core"
+			"LP072a Battery Pack"
+			"Liquid Nitrogen Cooler"
+			"Catalytic Ramscoop"
+			"Tactical Scanner"
+			"Cargo Scanner"
+			"Outfit Scanner"
+			"Asteroid Scanner"
+
+			"A120 Atomic Thruster"
+			"A125 Atomic Steering" 2
+			"Scram Drive"
+
+			"Emotional Baggage"
+			"Tear of Skade" 2
+	ship "Waverider" "Balancers' Madness"
+		"never disabled"
+		add attributes
+			"shields" 12000
+			"hull" 5000
+			"mass" 100
```

Comment:
```suggestion
	ship "Waverider" "Proofreader's Pain"
		"never disabled"
	ship "Waverider" "Art Critics' Regret"
		"never disabled"
	ship "Waverider" "Code Herders' Curse"
		"never disabled"
		add attributes
			"required crew" 2
	ship "Waverider" "Choreographers' Tears"
		"never disabled"
	ship "Waverider" "Adjuncators' Blood"
		"never disabled"
		outfits
			"Particle Cannon" 4
			"S-970 Regenerator"
			"Dwarf Core"
			"LP072a Battery Pack"
			"Liquid Nitrogen Cooler"
			"Catalytic Ramscoop"
			"Tactical Scanner"
			"Cargo Scanner"
			"Outfit Scanner"
			"Asteroid Scanner"

			"A120 Atomic Thruster"
			"A125 Atomic Steering" 2
			"Scram Drive"

			"Emotional Baggage"
			"Tear of Skade" 2
	ship "Waverider" "Balancers' Madness"
		"never disabled"
		add attributes
			"shields" 12000
			"hull" 5000
			"mass" 100
```
Are we still doing the difference in crew, or is the Code Herder ship crew change a joke in itself? If the latter then that's fine, since these others have some aspects.

Reply:
Code Herder's gimmick is that it'd disable itself due to lack of crew every 50% chance, so it's sort of a mix of both.

Reply:
yeah it's a bugy ship and it represents how we're undercrewed in that aspect